lighthouse-plugin-publisher-ads/audits/loads-gpt-from-sgdn.js

```diff
@@ -61,7 +61,7 @@ class LoadsGptFromSgdn extends Audit {
     const networkRecords = await NetworkRecords.request(devtoolsLog, context);
     const gptUrl = networkRecords.map((r) => new URL(r.url)).find(isGptTag);
     if (!gptUrl) {
-      return auditNotApplicable.NoGpt;
+      return auditNotApplicable.NoTag;
```

Comment:
Obviously more work is needed in this file. This is just to get it to continue to compile.

Reply:
Undoing this change